tox.ini

```
@@ -16,7 +16,7 @@ statistics = 1
 [tox]
 minversion=2.5
 envlist =
-    ipv6, pep8, py{27,35,36,37,38,39,py2,py3}-{selects,poll,epolls}, py38-openssl
+    ipv6, pep8, py{27,35,36,37,38,39,py2,py3}-{selects,poll,epolls,dnspython1}, py38-openssl
```

```suggestion
    dnspython1, ipv6, pep8, py{27,35,36,37,38,39,py2,py3}-{selects,poll,epolls}, py38-openssl
```

IMHO, it's sufficient to check against dnspython1 on any single Python version.